### PR TITLE
Generating sdk new namespace

### PR DESCRIPTION
**Pull request by mohabghanem**
## Description

Changing SDK Namespace and Adding new apis
https://github.com/Azure/azure-rest-api-specs/pull/3036
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@mohabghanem Please fix the failing tests

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Programmatic.sln [hunk 1]
```diff
@@ -1,50 +0,0 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Programmatic.sln]: I see `Programmatic.sln` is being replaced by `Authoring.sln` but the csproj names are still the same, please keep consistency, same goes for the package name
On a related note, is your intention to merely rename the directories or the name of the package too

**Review by dsgouda — CHANGES_REQUESTED** [review]
A few questions:
- Is this nuget package already published?
- If not, versioning must start with `1.0.0` in the csproj. 
- Is this a preview or a stable version? Please update package version accordingly. If this is a preview package the versioning must start from `0.9.0-preview`

**Comment by dsgouda** [comment]
@mohabghanem gentle reminder, PR shall be closed if comments are not addressed by end of next week

**Comment by mohabghanem** [comment]
@dsgouda this is a preview of a new nuget, I updated the versions accordingly and rebased the branch

**Comment by mohabghanem** [comment]
@dsgouda Are there any other required changes?

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime.Tests/Microsoft.Azure.CognitiveServices.LUIS.Tests.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.LUIS.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.LUIS.Tests</AssemblyName>
-    <Version>2.0.0</Version>
+    <Version>0.9.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime.Tests/Microsoft.Azure.CognitiveServices.LUIS.Tests.csproj]: Test projects are never published. Please set the version to `1.0.0`

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Runtime API.</Description>
-    <Version>2.0.0</Version>
+    <Version>0.9.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj]: If the package is not to be renamed, the preview version must be set to 2.0.1-preview, we cannot go back in versions

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring.Tests/Microsoft.Azure.CognitiveServices.LUIS.Authoring.Tests.csproj [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
-    <Description>Microsoft.Azure.CognitiveServices.LUIS-Programmatic.Tests Class Library</Description>
-    <AssemblyName>Microsoft.Azure.CognitiveServices.LUIS-Programmatic.Tests</AssemblyName>
-    <Version>2.0.0</Version>
+    <Description>Microsoft.Azure.CognitiveServices.LUIS-Authoring.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.LUIS-Authoring.Tests</AssemblyName>
+    <Version>0.9.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring.Tests/Microsoft.Azure.CognitiveServices.LUIS.Authoring.Tests.csproj]: These tests are never published, simply set the version to `1.0.0` and never update them

**Review by dsgouda — COMMENTED** [review]
It looks like some of the projects modified here do not have an `AssemblyInfo.cs` file. Please add one and set the assembly versions appropriately.
[Here](https://github.com/mohabghanem/azure-sdk-for-net/tree/c8e902b1307f6a6c73fa0a524078fd60986d746d/src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties) is an example.

**Comment by dsgouda** [comment]
@ronak25 and @sudhansu88 can you guys take a look at this too. Please confirm whether you are okay with the preview nuget packages being introduced here.

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj [hunk 5]
```diff
@@ -6,7 +6,7 @@
...
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.LUIS</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;LUIS;Language Understanding Intelligent Service;Language Understanding;REST HTTP client</PackageTags>
     <PackageReleaseNotes>This is a preview release of the Cognitive Services LUIS Runtime SDK. Included with this release is support for LUIS Prediction API.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj]: Please update the `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
@mohabghanem Please clean up the git commit history to just 1 commit and update the PR

**Comment by dsgouda** [comment]
Please squash your commits and update the PR.
@ronak25 Please approve these changes before we proceed.

**Review by sudhansu88 — CHANGES_REQUESTED** [review]
Taking a look.

File: src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj (hunk 5 above)
Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj]: Is this `LUIS Prediction API`?

**Review by sudhansu88 — COMMENTED** [review]
These all are related to LUIS and does not impact Bing API SDKs

**Comment by mohabghanem** [comment]
@dsgouda commits squashed

**Comment by shahabhijeet** [comment]
@mohabghanem can you complete the CLA process by visiting and signing in.
https://github.com/azurecla

**Comment by mohabghanem** [comment]
@shahabhijeet nothing happens when I click sign in on the page

**Comment by shahabhijeet** [comment]
@mohabghanem link updated
https://cla.opensource.microsoft.com/

**Comment by mohabghanem** [comment]
@shahabhijeet still, nothing happens when I click sign in, I tried different browsers

**Comment by mohabghanem** [comment]
@shahabhijeet any updates on the link?

**Comment by dsgouda** [comment]
@mohabghanem Please start an email thread regarding this.

**Comment by mohabghanem** [comment]
@dsgouda to whom should I send the email thread?

**Review by shahabhijeet — APPROVED** [review]

**Review by dsgouda — APPROVED** [review]

**Pull request merged**